bin/clean_az_dupl.sh

```diff
@@ -1,35 +1,36 @@
 #!/usr/bin/env bash
 
-# Related to bug fix: https://bugs.launchpad.net/ubuntu/+source/dpkg/+bug/1730627
```

Comment:
As explained above in the `.travis.yml` comments, given that having the azure-cli tool already installed is necessary in two different scripts, then I moved it out and put in only one place. That is, in the `.travis.yml` file.